src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import type { StoryObj, Meta } from '@storybook/react';
+
+import React from 'react';
+import { Badge } from './Badge';
+import Icon from '../Icon';
+
+export default {
+  // TODO-JL: Appropriate storybook folder?
+  title: 'Molecules/Messaging/Badge',
```

Comment:
Placed in Molecules/Messaging because it seemed most appropriate, advise if needed